src/components/User/Assets/Issuance/components/IssuanceForm.vue

```diff
@@ -25,6 +25,7 @@
             v-model="form.reference"
             :label="'issuance-form.lbl-reference' | globalize"
             @blur="touchField('form.reference')"
+            :maxlength="REFERENCE_MAX_LENGTH"
```

Comment:
It doesn't work. Props should be passed in kebab-case. `InputField` doesn't have `maxlength` props. Why you didn't check it?